src/Compilers/CSharp/Test/Emit3/Semantics/CollectionExpressionTests.cs

```diff
@@ -33373,8 +33389,10 @@ public void SpreadNullability()
                 string[] y2 = [.. x2];
                 """;
 
-            // https://github.com/dotnet/roslyn/issues/68786: We should check the spreads
-            CreateCompilation(src).VerifyEmitDiagnostics();
+            CreateCompilation(src).VerifyEmitDiagnostics(
+                // (5,19): warning CS8602: Dereference of a possibly null reference.
+                // string[] y1 = [.. x1];
+                Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "x1").WithLocation(5, 19));
```

Comment:
Parking comment here, but it relates to a test below. I think we can remove the comment in `SpreadNullability_SplitExpression`.